packages/circuit-ui/styles/style-mixins.spec.tsx

```diff
@@ -187,18 +187,19 @@ describe('Style helpers', () => {
   });
 
   describe('typography', () => {
-    type Size = keyof Theme['typography']['body'];
-    const snapshots: { size: Size; snapshot: string }[] = [
-      { size: 'one', snapshot: '"font-size:16px;line-height:24px;"' },
-      { size: 'two', snapshot: '"font-size:14px;line-height:20px;"' },
-    ];
-    it.each(snapshots)(
```

Comment:
Was running into the following error: "Multiple inline snapshots for the same call are not supported."

(see https://github.com/facebook/jest/issues/6636)